### ISSUE TITLE
Last update breaks JSON output in runJS

### ISSUE THREAD

**Issue by DigitalLeaves** [lukehaas/RunJS#167]
Hi,

Up until now, if you had a JSON object like these:
```
const a = {
  user: {
    locale: 'es_ES',
    first_name: 'IGNACIO',
    promo_code: null,
    status: 'ready',
    created: '2018-07-26T14:00:33.000Z',
    role: 'admin',
    is_admin: true,
    trusted: true,
    last_login: '2020-07-23T16:06:39.429Z',
    cert_auth_enabled: true
  },
  error: false,
  error_code: 'success'
}

console.log(JSON.stringify(a, null, 2))
```

It will print something beautiful like:
```
{
  "user": {
    "locale": "es_ES",
    "first_name": "IGNACIO",
    "promo_code": null,
    "status": "ready",
    "created": "2018-07-26T14:00:33.000Z",
    "role": "admin",
    "is_admin": true,
    "trusted": true,
    "last_login": "2020-07-23T16:06:39.429Z",
    "cert_auth_enabled": true
  },
  "error": false,
  "error_code": "success"
}
```

I used it a lot to prettify JSON output for our API documentation.

Now, it outputs this???

```
'{
' +
  '  "user": {
' +
  '    "locale": "es_ES",
' +
  '    "first_name": "IGNACIO",
' +
  '    "promo_code": null,
' +
  '    "status": "ready",
' +
  '    "created": "2018-07-26T14:00:33.000Z",
' +
  '    "role": "admin",
' +
  '    "is_admin": true,
' +
  '    "trusted": true,
' +
  '    "last_login": "2020-07-23T16:06:39.429Z",
' +
  '    "cert_auth_enabled": true
' +
  '  },
' +
  '  "error": false,
' +
  '  "error_code": "success"
' +
  '}'
```

Please, let me know if that's a new feature/setting that was introduced and how to get back to the previous behavior.

Thanks!

**Comment by lukehaas** [comment]
@DigitalLeaves thanks for raising this. You're right, the latter output does not look good.
The last release had some improvements for formatting and displaying outputs but unfortunately seems to have made this kind of output worse.
I'll look into improving this in the next release.

**Comment by lukehaas** [comment]
@DigitalLeaves this is now fixed in RunJS version 1.11.0

**Comment by DigitalLeaves** [comment]
Looking really good, and the rest of the features are very welcomed also, especially the search functionality.
Congratulations! 👏🏻
